starter-project-web-vue2/pages/andualem/_id.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div>
+    <nuxt-link style="text-decoration: none" :to="'/andualem/'">
+      <v-btn class="success">Back to Blogs</v-btn>
+    </nuxt-link>
+    <h2>Title: {{ title }}</h2>
+    <hr />
+    <h4>{{ content }}</h4>
+  </div>
+</template>
+
+<script>
+import axios from 'axios'
+export default {
+  data() {
+    return {
+      content: '',
+      title: '',
+    }
+  },
+  async created() {
+    const config = {
+      headers: {
+        Accept: 'application/json',
+      },
+    }
+    try {
+      const res = await axios.get(
+        `https://blog-app-backend.onrender.com/api/articles/${this.$route.params.id}`,
+        config
+      )
+      console.log(res)
+      this.content = res.data.content
+      this.title = res.data.title
+    } catch (err) {
+      console.log(err)
+    }
+  },
+}
+</script>
+<style>
+h2 {
+  font-family: Helvetica, Arial, sans-serif;
+  font-size: 15pt;
+  margin-top: 25px;
+  margin-bottom: 20px;
+}
+
```

Comment:
we don't need to add custom CSS here. we can do a similar thing using vuetify classes like `mb-2` for the margin and others too. check [here](https://vuetifyjs.com/en/styles/spacing)  for spacing and [here](https://vuetifyjs.com/en/styles/text-and-typography/) for the font styles